src/patterns/service-unavailable-pages/after-service-closes/index.njk

```diff
@@ -0,0 +1,26 @@
+---
+layout: layout-example.njk
+---
+
+<div class="govuk-width-container">
+
+  <div class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        <h1 class="govuk-heading-xl">Service unavailable</h1>
+        <p class="govuk-body-l">You cannot use the online service to renew your tax credits.</p>
+        <p class="govuk-body"><a class="govuk-link" href="https://www.gov.uk/contact-the-tax-credit-office">Contact the Tax Credit Helpline</a> to renew your tax credits by phone or letter.</p>
```

Comment:
Shall we use a `#` here instead of the real URL? I'm afraid we'll ruin statistics for `https://www.gov.uk/contact-the-tax-credit-office` if people click the link.

Reply:
+1 - it'll also open in the iframe, which probably isn't ideal.